docs/config.md

```diff
@@ -26,111 +26,158 @@ Sets the minimum amount of characters that need to be typed into the search dial
 
-Automatically expands the single field in a schema.
+### hideSchemaTitles
 
-### hideDownloadButton
+Hides the schema title next to to the type.
 
-Hides the 'Download' button for saving the API definition source file. **This setting does not make the API definition private**; it just hides the button.
+### jsonSamplesExpandLevel
 
-### hideHostname
+Sets the default expand level for JSON payload samples (response and request body). The default value is 2, and the maximum supported value is '+Infinity'. It can also be configured as a string with the special value `all` that expands all levels.
 
-If set to `true`, the protocol and hostname are not shown in the operation definition.
+_Default: 2_
 
-### hideLoading
+### maxDisplayedEnumValues
 
-Hides the loading animation. Does not apply to CLI or Workflows-rendered docs.
+Displays only the specified number of enum values. The remaining values are hidden in an expandable area. If not set, all values are displayed.
 
-### hideRequestPayloadSample
+### onlyRequiredInSamples
 
-Hides request payload examples.
+Shows only required fields in request samples.
 
-### hideOneOfDescription
+### sortRequiredPropsFirst
 
-If set to `true`, the description for `oneOf`/`anyOf` object is not shown in the schema.
+Shows required properties in schemas first, ordered in the same order as in the required array.
 
-### hideSchemaPattern
+### schemasExpansionLevel
 
-If set to `true`, the pattern is not shown in the schema.
+Specifies whether to automatically expand schemas in Reference docs. Set it to `all` to expand all schemas regardless of their level, or set it to a number to expand schemas up to the specified level. For example, `schemasExpansionLevel: 3` expands schemas up to three levels deep. The default value is `0`, meaning no schemas are expanded automatically.
 
-### hideSchemaTitles
+### scrollYOffset
 
-Hides the schema title next to to the type.
+Specifies a vertical scroll-offset.
+This setting is useful when there are fixed positioned elements at the top of the page, such as navbars, headers, etc.
 
-### hideSecuritySection
+Note that you can specify the `scrollYOffset` value in any of the following ways:
+- as a number - a fixed number of pixels to be used as the offset.
+- as a CSS selector - the selector of the element to be used for specifying the offset. The distance from the top of the page to the element's bottom is used as the offset.
+- a function (advanced) - a getter function. Must return a number representing the offset (in pixels).
 
-Hides the Security panel section.
+### showExtensions
 
-### hideSingleRequestSampleTab
+Shows specification extensions ('x-' fields). Extensions used by Redoc are ignored. The value can be boolean or an array of strings with names of extensions to display. When used as boolean and set to `true`, all specification extensions are shown.
 
-Hides the request sample tab for requests with only one sample.
+### sanitize
 
-### htmlTemplate
```

Comment:
Don't we still need this option? This is commonly used.

Reply:
Yes, you are right, it is very common. But `htmlTemplate` is not a Redoc CE option, it relates to `redocly-cli`. We moved it when was deprecated `redoc-cli` and moved code of build command to `redocly-cli`